# Solver summary of ceres solver
# see: http://ceres-solver.org/nnls_solving.html#solver-summary

Header header

string BriefReport
string FullReport